### PR TITLE
Added "Free Dictionary API" to Dictionaries

### PR DESCRIPTION
**Pull request by Sanobelk**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by fintzd — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -365,6 +365,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Chinese Character Web](http://ccdb.hemiola.com/) | Chinese character definitions and pronunciations | No | No | No |
 | [Chinese Text Project](https://ctext.org/tools/api) | Online open-access digital library for pre-modern Chinese texts | No | Yes | Unknown |
+| [Free Dictionary API](https://dictionaryapi.dev/) | Definitions, phonetics, pronounciations, parts of speech, examples, synonyms | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:368]: Title should not end with "... API". Every entry is an API here!

```suggestion
| [Free Dictionary](https://dictionaryapi.dev/) | Definitions, phonetics, pronounciations, parts of speech, examples, synonyms | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @Sanobelk :)

Can you resolve this review, please?

**Comment by Sanobelk** [comment]
> 
> 
> Thanks for your contribution @Sanobelk :)
> 
> Can you resolve this review, please?

Sure. I added the name because that's what the website specifically called it.

**Review by matheusfelipeog — APPROVED** [review]
Perfect. Thanks for updating it 😄

**Pull request merged**